src/pages/index.tsx

```diff
@@ -95,23 +95,11 @@ const Home: NextPage = () => {
     changeTokenApproved(Number(tokenAllowances[i]) > Number(fromDetails.amount));
   }, [fromDetails.symbol]);
 
-  useEffect(() => {
```

Comment:
Is there any reason that you removed the calculation of swap rate?

Reply:
see this commit: https://github.com/ZigZagExchange/mammoth_UI/pull/65/commits/703ae076b693387602b74635c4955ff0671d1779 We allready call getSwapAmount in 'predictSwapResult'. These both get called at the same time and request the same external information. Merged them to only call StarkNet once (less requests - they impose a rather low ratelimt)

Reply:
updated with https://github.com/ZigZagExchange/mammoth_UI/pull/65/commits/04ca13433eed157e2c764e1e90977e2d603eac2a